app_feup/lib/redux/action_creators.dart

```diff
@@ -199,6 +200,14 @@ ThunkAction<AppState> updateStateBasedOnLocalUserBusStops() {
   };
 }
 
+ThunkAction<AppState> updateStateBasedOnLocalUserPrintMovements() {
+  return (Store<AppState> store) async {
+    final AppPrintMovementsDatabase db = AppPrintMovementsDatabase();
+    final List movements = await db.printMovements();
+    store.dispatch(SetPrintMovementsAction(movements));
+  };
+}
+
```

Comment:
You should call this function on startup, in _loadLocalUserInfoToState_ on _load_info.dart_